.ci/openshift-ci/users.htpasswd

```diff
@@ -0,0 +1 @@
+user:{SHA}Et6pb+wgWTVmq3VpLJlJWWgzrck=
```

Comment:
if it's a dummy credential like 'user' it should be generated by the script and not stored in the repository
if it's a real credential, it needs to be stored with github secrets.